src/System Application/App/Telemetry/src/Logging/ALTelemetryScope.Enum.al

```diff
@@ -0,0 +1,27 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Telemetry;
+
+enum 8704 "AL Telemetry Scope"
+{
+    Access = Internal;
+
+    value(0; ExtensionPublisher)
+    {
+        Caption = 'Extension Publisher';
+        // map to TelemetryScope::ExtensionPublisher
+    }
+    value(1; Environment)
+    {
+        Caption = 'Environment';
+        // map to TelemetryScope::All
```

Comment:
I think that you should use the summary xml comment for such explanations instead of a simple comment here.
This would align with other parts of the system application

Reply:
Good catch, fixed